esx_service/utils/auth_data.py

```diff
@@ -664,7 +663,10 @@ def new_db(self):
         err = self.__create_default_tenant()
         if err:
             return err
-        err = self.__create_default_privileges()
+        err = self.__create_all_ds_privileges_for_default_tenant()
+        if err:
+            return err
+        err = self.__create_vm_ds_privileges_for_default_tenant()
```

Comment:
If "new_db()" returns with error, then the configuration is "NotConfigured", which is in "no access control" mode. So why we need to remove the "DEFAULT tenant"?